### ISSUE TITLE
Add compiled hex files

### ISSUE THREAD

**Issue by ea** [ea/srxe_cc1101#5]
Add precompiled , ready to flash, hex files and instructions on flashing them.

**Comment by RChadwick7** [comment]
Can these be updated to include the new power management enhancements?

**Comment by ea** [comment]
Good point. I was planning on playing around with these devices again one of these days. I'll refresh the binaries once i get around to it.
